### ISSUE TITLE
Unable to use Plugin-Lua with Prettier 2.3

### ISSUE THREAD

**Issue by jodysaliba** [prettier/prettier-vscode#1973]
**Prettier 2.3.0**

**Lua Plugin 0.0.1**

Just installed the Lua plugin for VSCode and began getting this error attempting to format my Lua files. Seems to work fine formatting the JSON files that I have in my project, but failes on the Lua ones. I couldn't find any other Issues regarding this searching "Lua" on the Issues page.

```
PS M:\OneDrive\WoW Addons\TooltipRatings> prettier --check "M:/OneDrive/WoW Addons/TooltipRatings/test.lua"
Checking formatting...
test.lua[error] test.lua: TypeError: locEnd is not a function
[error]     at isNextLineEmpty$e (C:\Users\myUser\AppData\Roaming\npm\node_modules\prettier\index.js:4600:44)
[error]     at C:\Users\myUser\AppData\Roaming\npm\node_modules\@prettier\plugin-lua\src\printer.js:410:7
[error]     at C:\Users\myUser\AppData\Roaming\npm\node_modules\prettier\index.js:15877:23
[error]     at AstPath.each (C:\Users\myUser\AppData\Roaming\npm\node_modules\prettier\index.js:15864:7)
[error]     at AstPath.map (C:\Users\myUser\AppData\Roaming\npm\node_modules\prettier\index.js:15876:10)
[error]     at printBody (C:\Users\myUser\AppData\Roaming\npm\node_modules\@prettier\plugin-lua\src\printer.js:384:8)
[error]     at printNoParens (C:\Users\myUser\AppData\Roaming\npm\node_modules\@prettier\plugin-lua\src\printer.js:28:14)
[error]     at Object.genericPrint [as print] (C:\Users\myUser\AppData\Roaming\npm\node_modules\@prettier\plugin-lua\src\printer.js:763:19)
[error]     at callPluginPrintFunction (C:\Users\myUser\AppData\Roaming\npm\node_modules\prettier\index.js:16169:21)
[error]     at mainPrintInternal (C:\Users\myUser\AppData\Roaming\npm\node_modules\prettier\index.js:16107:17)
All matched files use Prettier code style!
```

**Comment by ntotten** [comment]
Does the plugin work through the CLI? This looks like a bug I. The lau plug-in rather than the extension at first glance.

**Comment by jodysaliba** [comment]
The error in the OP was from the terminal in VSCode. I also tried in the Windows command prompt, and got the same error:
![image](https://user-images.githubusercontent.com/1418509/120367285-ae6f7380-c2de-11eb-975d-8ac95da1461a.png)

I thought I added this Issue to the plugin. But I just went and looked at the plugin issues and there is [one with my issue from January](https://github.com/prettier/plugin-lua/issues/20). Is the Lua Plugin not maintained anymore?
